concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1498,14 +1498,14 @@ Hale-1973-1798-1496	Role_and_Social_Structure-150	share	v	11b2.30	884	SHARE
 Hale-1973-1798-1497	Role_and_Social_Structure-151	own	v	11b2.31	684	OWN
 Hale-1973-1798-1498	Role_and_Social_Structure-152	serve a master	v	11b2.32	3648	SERVE (SOMEONE)
 Hale-1973-1798-1499	Role_and_Social_Structure-153	swear oath	v	11b2.33	1939	SWEAR
-Hale-1973-1798-1500	Role_and_Social_Structure-154	assemble	v	11b2.34		
+Hale-1973-1798-1500	Role_and_Social_Structure-154	assemble	v	11b2.34	3638	ASSEMBLE (GET TOGETHER)
 Hale-1973-1798-1501	Role_and_Social_Structure-155	elect	v	11b2.35		
 Hale-1973-1798-1502	Role_and_Social_Structure-156	vote	v	11b2.36		
 Hale-1973-1798-1503	Role_and_Social_Structure-157	collect taxes	v	11b2.37		
 Hale-1973-1798-1504	Role_and_Social_Structure-158	rob	v	11b3.01	2073	ROB (VERB)
 Hale-1973-1798-1505	Role_and_Social_Structure-159	steal	v	11b3.02	713	STEAL
 Hale-1973-1798-1506	Role_and_Social_Structure-160	plunder	v	11b3.03	3670	PLUNDER
-Hale-1973-1798-1507	Role_and_Social_Structure-161	attack	v	11b3.04		
+Hale-1973-1798-1507	Role_and_Social_Structure-161	attack	v	11b3.04	722	ATTACK
```

Comment:
Would unmap, since in concepticon this is a noun. https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L3673